packages/angular/src/generators/library/schema.json

```diff
@@ -181,6 +181,11 @@
       "type": "boolean",
       "default": false,
       "description": "Specifies if the component should have a selector or not. Disclaimer: This option is only valid when `--standalone` is set to `true`."
+    },
+    "flat": {
+      "type": "boolean",
+      "default": false,
+      "description": "When generating a library with `--standalone`, this option will ensure the generated standalone component is not placed in a subdirectory."
```

Comment:
Should we aim to keep a similar language for this one? In the other options we first state what the option is for and then, we have a disclaimer to mention that it only applies when `--standalone=true`.

Reply:
yes, we should. good idea